geomstats/geometry/rank_k_psd_matrices.py

```diff
@@ -38,6 +38,13 @@ def __init__(self, n, k, **kwargs):
         self.rank = k
         self.sym = SymmetricMatrices(self.n)
 
+    @property
```

Comment:
For consistency, shouldn't we override `_metric` if not defined at initialization and do not override the property at this level?

Reply:
It is not possible in that case because of circular/recursive initialisation: the metric here needs a `FiberBundle` which itself needs the space `RankKPSDMatrices` initialized.